test/conftest.py

```diff
@@ -495,23 +495,23 @@ def qtile(request, xephyr):
     with tempfile.NamedTemporaryFile() as f:
         sockfile = f.name
         try:
-            q = Qtile(sockfile, xephyr.display, request.config.getoption("--debuglog"))
-            q.start(config)
+            manager = TestManager(sockfile, xephyr.display, request.config.getoption("--debuglog"))
+            manager.start(config)
 
-            yield q
+            yield manager
         finally:
-            q.terminate()
+            manager.terminate()
 
 
 @pytest.fixture(scope="function")
-def qtile_nospawn(request, xephyr):
+def m_nospawn(request, xephyr):
```

Comment:
Anoter `m` - honestly can't tell whether this one is inentional, though.

Reply:
oh, no, that was a total miss by my using \<m\> as my search and replace. This means that this exists elsewhere too! Man, really not on the ball today. I will replace m_nospawn everywhere.